backlog/routes/ProjectRoute.ts

```diff
@@ -18,10 +18,15 @@ class ProjectRoute {
   getProjectById(endpoint: string) {
     return router.get(`${endpoint}`, this.controller.getProjectById)
   }
+
+  deleteProjectById(endpoint: string) {
+    return router.delete(`${endpoint}`, this.controller.deleteProjectById)
+  }
 }
 
 const routes = new ProjectRoute(projectController)
 routes.getProjects('/')
 routes.getProjectById('/:id')
+routes.deleteProjectById('/delete/:id')
```

Comment:
```suggestion
routes.deleteProjectById('/:id')
```
it's ok to remove delete because we are using `router.delete` as an identifier